### PR TITLE
Currently cannot setup_oscap upstream as pkgs only in downstream.

### PR DESCRIPTION
**Pull request by kbidarkar**
Currently cannot setup_oscap upstream as pkgs are only available in downstream.

### CODE REVIEW

File: automation_tools/__init__.py [hunk 1]
```diff
@@ -1000,11 +1000,13 @@ def product_install(distribution, create_vm=False, certificate_url=None,
 
     if distribution.startswith('satellite6'):
         execute(setup_default_docker, host=host)
-        execute(setup_oscap, host=host)
         execute(katello_service, 'restart', host=host)
         execute(install_puppet_scap_client, host=host)
         execute(setup_foreman_discovery, host=host)
 
+    if not distribution.endswith('upstream'):
```
Comment (elyezer) [inline automation_tools/__init__.py]: Please indent this if block to be inside the satellite distribution as we use this for sam too.

**Comment by elyezer** [comment]
ACK, pending comment.

**Comment by Ichimonji10** [comment]
ACK pending @elyezer's comment.

**Comment by Ichimonji10** [comment]
Comment addressed. Travis passes. Merging.

**Pull request merged**